SPECS/bind/bind.spec

```
@@ -536,7 +536,10 @@ fi;
 %{_mandir}/man1/named-nzd2nzf.1*
 
 %changelog
-* Tue Feb 11 2025 Andrew Phelps <anphel@microsoft.com> - 9.20.5-4
+* Mon May 26 2025 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 9.20.9-1
+- Auto-upgrade to 9.20.9 - for CVE-2025-40775
+
+* Tue Mar 11 2025 Andrew Phelps <anphel@microsoft.com> - 9.20.5-4
```

Why was the date changed?

This needs to be:
```suggestion
* Tue Feb 11 2025 Andrew Phelps <anphel@microsoft.com> - 9.20.5-4
```